glossary/LODASH.md

```diff
@@ -1,3 +1,5 @@
 # Lodash
 
-[Lodash](https://lodash.com/docs) section to be completed.
+[Lodash](https://lodash.com) is a small [modulized](https://www.npmjs.com/browse/keyword/lodash-modularized) library which provides bunch of helpers to work with array, string,... much simpler. Every single helper can be required and used itself without requiring the whole library. Additional, it makes Javascript code look elegant and more efficient by offering functional style, method chaining and [more](https://lodash.com/#features).
```

Comment:
I’d say “which provides **a** bunch of helpers to work with **JavaScript primitives in a much more simpler way**.” What do you think?